Updated graphics

jdgregson opened the pull request:
I'm personally not a fan of the Duck Hunt-style graphics. Beamgun is an effective countermeasure to a legitimate security issue, and it should take its role seriously. As such, I have created a fresh, modern set of icons and banners. Feel free to pull them into the project. 

![beamgunapp-1](https://user-images.githubusercontent.com/3778841/50130629-5722d900-0233-11e9-9d31-a890933ea16f.png)
![beamgunapp-2](https://user-images.githubusercontent.com/3778841/50130635-5b4ef680-0233-11e9-95c3-2655ce5f72d8.png)

jdgregson committed:
Added new icons and banners

jdgregson committed:
Set banner to render in high quality

JLospinoso commented:
@jdgregson Thanks for your PR! I think it would be a nice option for the user to have these modern-looking graphics. The default should be the current theme.

How would you propose to expose the option to the user?

jdgregson commented:
I've implemented Mouse blocking and added an option to lock on mouse, and the interface is getting a little busy. Most likely I'll be moving all of the check boxes to a little popup menu, so I could potentially include an option for icon themes. One issue with that though is that changing themes won't change the icon in the start menu. I didn't like the dog there, so I built the installer and reinstalled. 

Maybe I'll add all of my new features to something like Beamgun 2 and backport big changes (like Mouse blocking, password protection, etc.) via individual PRs.

jdgregson committed:
Added support for alerting on mouse insertions

jdgregson committed:
Fixed settings

jdgregson committed:
Added settings to support graphics themes

This commit does the following:
- Adds a "Modern" and "DuckHunt" graphics folder and moves graphics accordingly.
- Adds settings to support changing between graghich files.
- Adds Mouse alert icons to support the upcoming MouseWatcher feature.
- Sets DuckHunt as the default graphics.

jdgregson committed:
Set exe icon back to the dog

jdgregson commented:
I have implemented the following changes in commit b156fb5:
- Adds a "Modern" and "DuckHunt" graphics folder and moves graphics accordingly.
- Adds settings to support changing between graphic files.
- Adds Mouse alert icons to support the upcoming MouseWatcher feature.
- Sets DuckHunt as the default graphics.

jdgregson committed:
Merge pull request #1 from jdgregson/MouseWatcher

Merging MouseWatcher

jdgregson committed:
SetGraphicsMouseAlert now loads the correct graphics
